extensions/wikidata/tests/src/org/openrefine/wikidata/operations/PerformWikibaseEditsOperationTest.java

```diff
@@ -64,7 +64,7 @@ public void testConstructor() {
     @Test
     public void testLoadChange()
             throws Exception {
-        String changeString = "newItems={\"qidMap\":{\"1234\":\"Q789\"}}\n" + "/ec/\n";
+        String changeString = "newEntities={\"qidMap\":{\"1234\":\"Q789\"}}\n" + "/ec/\n";
```

Comment:
Let's keep the serialization as it was here:
```suggestion
        String changeString = "newItems={\"qidMap\":{\"1234\":\"Q789\"}}\n" + "/ec/\n";
```